ed_sensor_integration/tools/fitter_viz_live.cpp

```diff
@@ -0,0 +1,114 @@
+#include <ed/entity.h>
+#include <ed/world_model.h>
+
+#include <ed/kinect/fitter.h>
+#include <ed/kinect/image_buffer.h>
+
+#include <geolib/math_types.h>
+
+#include <opencv2/highgui/highgui.hpp>
+
+#include "ed_sensor_integration/tools/fitter_viz.h"
+#include "ed_sensor_integration/tools/snapshot.h"
+
+
+/**
+ * @brief usage, print how the executable should be used and explain the input
+ */
+void usage()
+{
+    std::cout << "Usage: ed_fitter_live WORLDMODEL_NAME  ENTITY_ID [RGBD_TOPIC]" << std::endl;
+}
+
+/**
+ * @brief main executable to visualise the fitting process of live images.
+ * @param argc: should be 4
+ * @return
+ */
+int main(int argc, char **argv)
+{
+    if (argc < 3 || argc > 5)
+    {
+        usage();
+        return 1;
+    }
+
+    std::string model_name = argv[1];
+
+    ed::WorldModelPtr world_model;
+    try
+    {
+        world_model = ed::loadWorldModel(model_name);
+    }
+    catch (ed::ModelNotFoundException e)
+    {
+        std::cerr << "World model '" << model_name << "' could not be loaded." << std::endl;
+        std::cerr << e.what() << std::endl;
+        return 1;
+    }
+
+    std::string entity_id = argv[2];
+    ed::EntityConstPtr e = world_model->getEntity(entity_id);
+    if (!e)
+    {
+        std::cerr << "Entity '" << entity_id << "' could not be found in world model '" << model_name << "'." << std::endl;
+        return 1;
+    }
+
+    std::string topic;
+    if (argc > 3)
+    {
+        topic = argv[3];
+        std::cout << "using topic: " << topic << std::endl;
+    }
+    else
+    {
+        topic = "head_rgbd_sensor/rgbd";
```

Comment:
I suggest to default to rgbd, as this is too much focused on HERO. While everywhere in our code we default to rgbd.

Reply:
While I agree we should try to keep thing generic where possible I also would like this tool to be accessible (BEP student level of accessibility) and since for the next couple of years the topic will always be "/hero/head_rgbd_sensor/rgbd" I wonder if this is a case of "You aren't gonna need it"

Reply:
BEP students should still learn ROS and topic remapping, if you ask me. Otherwise create a function which does the trick. You still need to run this one in a NS, to get shit working.

Reply:
removed the default and added a more extensive usage() function for students. I dont think they need to learn ROS and topic remapping. They need to deal with enough(me) as it is.